docs/developer-reference/contracts/vault-api.md

```diff
@@ -179,6 +179,24 @@ This `Vault` function swaps tokens based on provided parameters. All parameters
 |---|---|---|
 | params  | VaultSwapParams  | Parameters for the swap operation  |
 
+The `VaultSwapParams` are defined as
```

Comment:
```suggestion
[VaultSwapParams](https://github.com/balancer/balancer-v3-monorepo/blob/main/pkg/interfaces/contracts/vault/VaultTypes.sol#L277:L295) is defined as:
```

Reply:
Even though the name is technically pluralized, I think it's more natural to say "<structName> is defined as...", and we might as well include a link to the span and the associated comment.

Here and elsewhere.

Reply:
Thanks for you suggestions.